proseco/catalog.py

```diff
@@ -54,11 +54,16 @@ def _get_aca_catalog(obsid=0, att=None, man_angle=None, date=None,
     aca.fids = get_fid_catalog(detector=detector, sim_offset=sim_offset,
                                focus_offset=focus_offset, acqs=aca.acqs,
                                stars=aca.acqs.meta['stars'], print_log=print_log)
+    # Set the fid thumbs up to true for now as I'm not sure what to check
```

Comment:
Thumbs up for fid can be if it `len(fids) == fids.meta['n_fid']`.  Set it in `get_fid_catalog`.